src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -122,6 +123,17 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 
 		this._register(this._onAddConnectionProfile);
 		this._register(this._onDeleteConnectionProfile);
+
+		const seconds = 600;
+
+		setInterval(() => {
```

Comment:
Make sure you add comments for stuff like this explaining why this is necessary - it's definitely not clear from just the code itself why you need to set up this interval.